numba/core/boxing.py

```diff
@@ -1105,3 +1105,99 @@ def unbox_typeref(typ, val, c):
 @box(types.LiteralStrKeyDict)
 def box_LiteralStrKeyDict(typ, val, c):
     return box_unsupported(typ, val, c)
+
+
```

Comment:
Should probably reference the original source of this patch? https://github.com/numba/numba/issues/4499#issuecomment-1063138477